### PR TITLE
new Presence YuGiOh topdecks and wikiHow

### PR DESCRIPTION
**Pull request by sunx2**
yugioh topdecks is a top deck database

### CODE REVIEW

File: wikiHow/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        "id": "270125856679002113"
+    },
+    "service": "wikiHow",
+    "description": {
+        "en": "wikiHow - How to do anything!",
+        "ro": "wikiHow - Cum să faci orice!",
+        "pt_BR": "wikiHow - Як зробити що-небудь!",
+        "fr": "wikiHow - Comment faire n'importe quoi!",
+        "uk": "wikiHow - Як зробити що-небудь!",
+        "nl": "wikiHow - Як зробити що-небудь!"
```
Comment (Bas950) [inline wikiHow/dist/metadata.json]: The translations arent right, if you dont know the translation of it remove the whole line

Comment (sunx2) [reply wikiHow/dist/metadata.json]: done!

File: wikiHow/dist/metadata.json [hunk 2]
```diff
@@ -6,11 +6,6 @@
     "service": "wikiHow",
     "description": {
         "en": "wikiHow - How to do anything!",
```
Comment (Bas950) [inline wikiHow/dist/metadata.json]: Remove "," behind english translation

Comment (sunx2) [reply wikiHow/dist/metadata.json]: ok

Comment (sunx2) [reply wikiHow/dist/metadata.json]: done

**Pull request merged**